docs/vocascan-frontend/themes.md

```diff
@@ -0,0 +1,29 @@
+# Themes
+
+You can include custom themes in your vocascan-frontend installation by mounting a directory containing your themes into ``/usr/share/nginx/html/themes``.
+A theme has to define all the variables defined by the [default themes](https://github.com/vocascan/vocascan-frontend/tree/main/public/default-themes).
+To create a new theme, you can therefore copy one of the default themes and adjust the values.
+
+### Example
+
+Folder structure:
+
+```
+|-- themes
+|   |-- my-theme.css
+```
+
+Example theme (``my-theme.css``):
+
+```css
+:root {
+    --background-color: #fff;
+    /* Further configuration... */
+}
+```
+
+Mounting the ``themes/`` directory:
+
+```bash
+docker run -v ./themes:/usr/share/nginx/html/themes ...
```

Comment:
afaik this command wont work. You cant use relative paths via docker run. You need something like `$(pwd)/themes` but I suggest changing that to an docker-compose diff what you need to change.

Reply:
This does work, I used it for testing

Reply:
ok. I havent said anything :D - but can we still change that to docker-compose?

Reply:
Done